go.mod

```diff
@@ -1,8 +1,8 @@
 module github.com/transparency-dev/formats
 
-go 1.21.0
+go 1.22.0
 
-toolchain go1.22.3
+toolchain go1.22.5
```

Comment:
```suggestion
toolchain go1.22.7
```

Reply:
Unfortunately, we can't bump to 1.22.7 yet as TamaGo is only at 1.22.6 currently.

Reply:
Ah right, it will take a few days to catch up.